workspaces/copilot/plugins/copilot-common/src/index.ts

```diff
@@ -176,3 +178,173 @@ export interface TeamInfo {
    */
   name: string;
 }
+
```

Comment:
Not going to comment on each line but can we please use camel case for name in this file?

Reply:
This existing file already uses underscore_case, I don't mind adjusting it if the convention has changed or was wrong from the start. 👍🏻

Reply:
Yeah, I had not really looked at this file closely, it does seem to use that format, just not something I see in the rest of this plugin and not something I've generally seen in the Backstage eco-system. For this specific file let's leave it but elsewhere we should adjust it.